src/DynamoCoreWpf/Views/PackageManager/Controls/NumericUpDownControl.xaml.cs

```diff
@@ -141,10 +141,11 @@ private void inputField_PreviewTextInput(object sender, TextCompositionEventArgs
         // Handles the direct input of text (via keyboard)
         private void inputField_TextChanged(object sender, TextChangedEventArgs e)
         {
-            if (Int32.TryParse(inputField.Text, out int value))
+            var val = Int32.TryParse(inputField.Text, out int value);
+            if(val || value == 0)
```

Comment:
Isn't this check _always_ true? If the parsing succeeded then val is true. If it didn't, then value has its default value of 0.

Reply:
Yes, and now we are assigning that default value in cases where `TryParse` is false.

Reply:
ok, but why make an if statement that's always true?

Reply:
makes sense, updated.